cmake/CMakeInitializeConfigs.cmake

```diff
@@ -0,0 +1,40 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
```

Comment:
https://cmake.org/licensing/ link mentioned after "or" directs to the Copyright.txt file. I have simply copied this file from here https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/CMakeInitializeConfigs.cmake without any modifications. Thanks.

Reply:
update: I have reached out to our legal team for this as well.

Reply:
In such cases, it is better to avoid confusion and have legal clearance.  (1) I don't know if `Copyright.txt` file actually _requires_ you to copy itself along with the code. It is often the case with open source licenses: failing to provide the copy of the _license_ file violates the license.  (2) There is a `COPYRIGHT` file in the root directory which has a _different_ copyright and license, and may be confused with what is mentioned here.  (3) If there is a way to add code with external copyright, I want my code to have my copyright ;)

Reply:
This is the first point of the [Copyright.txt](https://gitlab.kitware.com/cmake/cmake/blob/master/Copyright.txt) file you did not copy
```
CMake - Cross Platform Makefile Generator
Copyright 2000-2018 Kitware, Inc. and Contributors
All rights reserved.

* Redistributions of source code must retain the above copyright
  notice, this list of conditions and the following disclaimer.
```

I don't see the "above copyright notice" nor the "following disclaimer" in their code you essentially redistribute.  I would claim this violates the license.